harness/determined/experimental/_native.py

```diff
@@ -68,17 +68,7 @@ def _submit_experiment(
     config["internal"]["native"] = {"command": _set_command_default(context_path, command)}
```

Comment:
You also need to move this whole block to `_construct_experiment` since the experiment using native API should have an `internal` field in the experiment config.

Reply:
`submit` acts like it is submitting a standard experiment, so it does not need to have the native internal command. This is why setting the command is part of the `_submit_experiment` that native uses, but it is not present in `submit`.